addons/account/tests/test_account_payment_register.py

```diff
@@ -1268,9 +1271,9 @@ def test_register_payment_invoice_foreign_curr_payment_comp_curr(self):
             'currency_id': self.other_currency.id,
             'invoice_line_ids': [Command.create(
                 {'product_id': self.product_a.id,
-                'price_unit': 1200.0,
-                'tax_ids': [],
-            })],
+                 'price_unit': 1200.0,
```

Comment:
i get the diff but if ruff is not complaining let's not put useless diff here :smile: